features/health-check.feature

```diff
@@ -14,13 +14,26 @@ Feature: Health Check
     Then the output includes 'There is an issue with your Conjur configuration.'
     And the exit status should be 1
 
+  Scenario: Policy Branch is Set
+    When I run the health check script with env CONJUR_FOLLOWER_URL=$CONJUR_APPLIANCE_URL CONJUR_POLICY=$CONJUR_POLICY
+    Then the exit status should be 0
+
+  Scenario: Policy Branch is Not Set
+    When I run the health check script with env CONJUR_FOLLOWER_URL=$CONJUR_APPLIANCE_URL CONJUR_POLICY=
+    Then the exit status should be 0
+
+  Scenario: Policy Branch Doesn't Exist
+    When I run the health check script with env CONJUR_FOLLOWER_URL=$CONJUR_APPLIANCE_URL CONJUR_POLICY=nonexistent_policy
```

Comment:
is the follower url needed on this line, or would this test case still be valid if you dropped it?

Reply:
Not needed - removed! 👍